apps/veil/package.json

```diff
@@ -102,5 +102,9 @@
     "typescript": "5.5.3",
     "typescript-eslint": "^8.17.0",
     "vitest": "^1.6.0"
+  },
+  "optionalDependencies": {
+    "lightningcss-darwin-arm64": "^1.30.1",
+    "@tailwindcss/oxide-darwin-arm64": "^4.1.11"
```

Comment:
question: what for?

Reply:
Lightningsccs doesn't work without this on my m1 Mac for whatever reason